### DISCUSSION TITLE
What vehicle attributes are important to track for car sharing?

### DISCUSSION THREAD

**Discussion by jfh01** [openmobilityfoundation/mobility-data-specification#657] [Car sharing]
It goes without saying that car sharing uses different vehicle types than micromobility. While some characteristics may be shared (propulsion type, for example), cars can have a variety of properties that scooters do not, and which may be relevant to policymakers. For example, passenger capacity, accessibility, child seat configuration, safety features, make/model/year.

As we look at adding car sharing support to MDS (#640), what vehicle characteristics are most important to capture in the data? How will these characteristics be useful to policymakers?

**Comment by jfh01** [comment]
Some initial discussion can be found here: https://github.com/openmobilityfoundation/mobility-data-specification/issues/640#issuecomment-842654240

**Comment by avatarneil** [comment]
I'd really like to make sure that we can capture accessibility options specifically, and I _think_ that we could probably enumerate them too. Most prominent accessibility option I can think of from my work around Taxi/TNC is `wheelchair_accessible`; but, noting that there are probably many other options we could want to include, I think we ought to make the property an `accessibility_options` list, as opposed to a boolean for `wheelchair_boarding: true` or something like that.
